### PR TITLE
Create tlg2200.tlg00549.opp-grc1.xml

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Added in work title
Added in language tagging
Updated publication date
Corrected bibliographic data

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed         |   Status |
|:----------------|---------:|
| `coverage`      |    -0.07 |
| `nodes_count`   |  -106    |
| `texts_passing` |    -1    |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    -6772 |

## Units

| Changed                                                 | Status   |
|:--------------------------------------------------------|:---------|
| `./data/tlg2200/tlg00549/tlg2200.tlg00549.opp-grc1.xml` | Failing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/1483)*

File: data/tlg2200/tlg00549/tlg2200.tlg00549.opp-grc1.xml [hunk 1]
```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
-<TEI xmlns="http://www.tei-c.org/ns/1.0">
```
Comment (sonofmun) [inline data/tlg2200/tlg00549/tlg2200.tlg00549.opp-grc1.xml]: You deleted the `<TEI>` opening tag and, thus, the file does not parse.

Comment (AlisonBabeu) [inline data/tlg2200/tlg00549/tlg2200.tlg00549.opp-grc1.xml]: Well at least I'm consistent I'll fix this too.

**Comment by capitains-hook** [comment]
*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/1550)*

**Pull request merged**